sdks/keyvault/Cargo.toml

```diff
@@ -26,6 +26,8 @@ serde = { version = "1.0", features = ["derive"] }
 getset = "0.1"
 oauth2 = { version = "3.0.0-alpha.9", features = ["reqwest-010", "futures-03"], default-features = false}
 azure_auth_aad = { version = "0.1", path = "../auth_aad" }
+azure_sdk_core = { path = "../core", version = "0.43.5" }
+async-trait = "0.1"
```

Comment:
The pull request looks great! It looks like this is a dev-dependency instead. It is only used in tests.

Reply:
thx, fixing...